contracts/modules/interfaces/ENSIntegration.sol

```diff
@@ -1,26 +1,9 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
 // SPDX-License-Identifier: Apache-2.0
 
-/******************************************************************************
- * Copyright 2020 IEXEC BLOCKCHAIN TECH                                       *
- *                                                                            *
- * Licensed under the Apache License, Version 2.0 (the "License");            *
- * you may not use this file except in compliance with the License.           *
- * You may obtain a copy of the License at                                    *
- *                                                                            *
- *     http://www.apache.org/licenses/LICENSE-2.0                             *
- *                                                                            *
- * Unless required by applicable law or agreed to in writing, software        *
- * distributed under the License is distributed on an "AS IS" BASIS,          *
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
- * See the License for the specific language governing permissions and        *
- * limitations under the License.                                             *
- ******************************************************************************/
-
 pragma solidity ^0.6.0;
 pragma experimental ABIEncoderV2;
 
-
-interface ENSIntegration
-{
-	function setName(address ens, string calldata name) external;
+interface ENSIntegration {
+    function setName(address ens, string calldata name) external;
```

Comment:
de we want to keep input argument name ?

Reply:
I didn't touch the content of the contracts. The goal is to format without doing any manual changes (except the copyright update).